data/json/snippets/snippets.json

```diff
@@ -1803,6 +1803,54 @@
       "\"Lisa always was a fan of sports.  Thank God these Global Athletics Games in Harran started some time before the outbreak, so she was able to leave New England before it was quarantined.  I can only hope there are no zombies, as we do have here, in the city she flew to.\""
     ]
   },
+  {
+    "type": "snippet",
+    "category": "note",
+    "text": ["\"Whoever finds this, listen to these words. This is what has kept me alive this long. If I’m not back in a day though, this isn’t enough. Burn some of those walking corpses in my memory. \n1. Zombies hate noise, they'll try to kill you if you make it, but enough noise will kill THEM\n2. The walking plant things are allergic to ammonia\n3. Zombies can’t smell you through smoke!\n4. Fire kills EVERYTHING\n5. Zombies can't smell OR see you if you're high on meth! They think you'r one of them!\n If you meet someone out there teach them what you know. It’s important that we preserve information.\n 7. If someone asks you for your hard earned equipment though, just give them a lit stick of dynamite\n 8. Watch out for wolves. Keep an eye out for cougars. Don't fuck with bears.\n 9. A water purifier is the best thing you can find. Clean water is precious.\n 10. Children zombies LOVE candy\n\""
```

Comment:
I stand by my earlier statement: this is too much for any single survivor to put in a note at the start of the game, and would be better suited to a couple notes, or held off until we can adjust spawning to make it arrive later, with better reading options.

Reply:
@I-am-Erk 

If you would like to implement said feature or have someone else implement said feature, and then add this lore in, please don't hesitate to do so. 

Otherwise feel free to remove it from this pull

Reply:
Removed.